content/blog/2017/2017-01-12-scm-api-2.0-release.adoc

```diff
@@ -0,0 +1,216 @@
+---
+:layout: post
+:title: SCM API turns 2.0 and what that means for you
+:tags:
+- development
+- plugins
+:author: stephenc
+---
+
+=== TL;DR
```

Comment:
Remove this header.  This is the introduction.  You have the TL;DR in a tip below, that is enough.